hubspot3/tickets.py

```diff
@@ -78,3 +90,185 @@ def get_all(self, limit: int = -1, **options) -> list:
+            ticket_id = int(ticket_id)
+            properties = ticket_history["properties"]
+            # Match the information of changes with each change
+            changes = changes_by_id[ticket_id]
+            for change in changes:
+                if change["changeType"] == TicketsClient.Recency.DELETED:
+                    # Deleted changes have no properties
+                    continue
+                change["changes"]["changedValues"] = {}
+                for changed_variable in change["changes"]["changedProperties"]:
+                    versions = properties[changed_variable]["versions"]
+                    # Get the smallest time diff between change and history
+                    time_diffs = [abs(version["timestamp"] - change["timestamp"])
+                                  for version in versions]
+                    min_time_index = min(range(len(time_diffs)),
+                                         key=time_diffs.__getitem__)
+                    if time_diffs[min_time_index] <= max_time_diff_ms:
+                        value = versions[min_time_index]["value"]
+                        change["changes"]["changedValues"][changed_variable] = value
+                    else:
+                        get_log(__name__).warning(f"Unable to find value for {changed_variable}"
+                                                  " within the time range for ticket"
+                                                  f" {ticket_id}")
+        # Finally lets return only the changes as a list
+        return [change for changes in changes_by_id.values() for change in changes]
+
+    def get_recently_modified_as_generator(self, limit: int = -1, time_offset: int = 0,
+                                           ticket_id_offset: int = 0) -> List[dict]:
+        """
+        get recently modified and created tickets, adding a field in changes-changedValue with
+        the value of each change
+        the returned value is done with yield at each page (max 1000 changes)
+        :see: https://developers.hubspot.com/docs/methods/tickets/get-ticket-changes
+        """
+        return self._get_recent(TicketsClient.Recency.MODIFIED, limit=limit,
+                                time_offset=time_offset, ticket_id_offset=ticket_id_offset)
+
+    def get_recently_modified(self, limit: int = -1, time_offset: int = 0,
+                              ticket_id_offset: int = 0) -> List[dict]:
+        """
+        get recently modified and created tickets, adding a field in changes-changedValue with
+        the value of each change
+        returned as a list of all changes
+        :see: https://developers.hubspot.com/docs/methods/tickets/get-ticket-changes
+        """
+        generator = self.get_recently_modified_as_generator(limit=limit, time_offset=time_offset,
+                                                            ticket_id_offset=ticket_id_offset)
+        return list(itertools.chain.from_iterable(generator))
```

Comment:
I commented in the `yield from` previously and looks like you rolled your own way to avoiding the problem I mentioned lol.